### ISSUE TITLE
LB health probes configuration issue

### ISSUE THREAD

**Issue by enovationIT** [Azure/Moodle#131]
At present both rules for ports 80 and 443 are using the same probe monitoring port 80, so if nginx fails like in #130 the load balancer is still trying to send traffic to 443 which nothing is listening at, to fix another probe should be added for port 443 and associated with the lb rule

**Comment by hosungsmsft** [comment]
Good point. I suppose this will be a simple fix (addition) in the template at https://github.com/Azure/Moodle/blob/master/nested/network.json#L250 . Again, a PR on this issue will be greatly welcome and appreciated.
